### ISSUE TITLE
Story

### ISSUE THREAD

**Issue by bojidar-bg** [KOBUGE-Games/ringed#19]
It would be nice to have at least some base story, so we can develop features connected with it.

**Comment by mamarilmanson** [comment]
I think story is best when it's confidential... XD

**Comment by erik90mx** [comment]
well I think we may define the "kind" of game first? @bojidar-bg  @mamarilmanson 
something like mood, mechanics, art etc.

**Comment by erik90mx** [comment]
Well after a while speaking in the phone with a writer that I know, we manage to get a possible abstract of the story:  

**STORY 1**
"**Introduction:**
The laboratory SpiderMyth is especially focus in the harvest of spider's venom to make antidotes, they has to their disposal more than 30 000 different spiders species, so is the leading exponent of this industry,however other types of experiments are also made in the laboratory, experiments that are government sponsored.

The personal that work in the installations, have to sign a document of confidentiality before work in this place. This way our character knows that something seems wrong inside the laboratory, however he never gave it the importance that the issue deserves.

One night like any other, the laboratory was very quiet, It was not until one of the alarms in underground laboratories begun to rang, that all the personal was in alert state, it was the first time in 15 years that something like that happens. They tried to calm down thinking that there was a failure in the security system, two of the guards who were finishing their shift decided to look down the place and know that was what had caused the alarm. Things went well for the first five minutes when they went down the elevator, but after joked about it for a while the elevator doors were opened, and in front of them, they found a swarm of giant spiders !, they managed to alert his colleagues, but for themselves it was too late.

The protagonist, realizes that the elevator was going up, but he could not communicate with his colleagues, when he looks security cameras, he saw with horror the bodies of their colleagues lying on the floor. The protagonist activated silent alarm and the building began to hold in quarantine, while one of his colleagues ran to leave the place, although it was impossible; The containment doors were faster than him, now both were stuck and not knowing who or what was their enemy

**Here the game starts**:exclamation:

**Story 1 Character Profile**
- **Sex:** Masculine
- **Race/species:** Human
- **Ethnicity:** American
- **Age:** 43
- **How old they appear:** 51
- **Occupation:** Night Security Guard
- **A Little Back Story:** He was in the army and participated in the war, but after a leg injury had to retire, when he returned home to live with his wife, she left him. He has worked 15 years in the company that the government recommends. He is a quiet person, a bit boring, he does not care about the life and is misaligned. He never had children and keeps unstable relationships with different types of women.

_"Heroes do not need to be perfect to be heroes"_ "

Hope you like it, feel free to modify it.

**Comment by bojidar-bg** [comment]
@erik90mx ~ that doesn't seem like a bad story... just we should think if we will make cinematic effects, will we keep all "images" in the same top-down style, etc.

And yeah, it badly needs a continuation, and an adaption for multiplayer..

**Comment by ObaniGemini** [comment]
@erik90mx : this looks like a classic action film story :/. I think we should look to some more abstract or original scenario, if there's one. Or just make a simple funny scenario :)

**Comment by alketii** [comment]
Very nice story :)

**Comment by ObaniGemini** [comment]
I will explain a little bit more what I mean : 
An abstract story : a story that goes out from usual stories, or that is not told, like in Braid or Perdition ( http://www.newgrounds.com/portal/view/641030 )
A simple funny scenario : Bread has a simple and funny scenario : you are a bread that wants to be toasted. Here it could be something like : The alien king from uranus wants to know the beer formula nd sends his army of giant spiders too discover it ^^

**Comment by erik90mx** [comment]
Well I like bizarre stories too, time ago I make a game called Waffles so... I got the idea.
I will try to explain those points to the writer, and If we get other abstract I will submit it (I need an editor to look the grammar, some grammar nazi over here?) 
I want to see the opinion of the others too :)

**Comment by erik90mx** [comment]
What is planning after the building lock of the protagonist (if we stay with that abstract) is this more or less:

Facts:
-There is no electricity in the building and the oxigen in limited.
-There is no external communication.

First Missions:
-Take a oxigen mask.
-Activate the building somnifer.
-Activate the energy in the building.
-Go out of the building.
-Comunicate about the issue to the CEO

Weapons/Consumible:

**_Start**_
-9mm Gun
-Truncheon
-Lighter
-Lantern

**_Found**_
-Tube
-Extintor
-Flammable liquids
-Fireman axe

When the protagonist is out of the building we can get more background story and weapons.
But I need to hear your feedbak with this issue, if you are Ok with that story, I will ask to the writer to work it more, if not I will ask for other story :)

@KOBUGE-Games/owners

**Comment by Etiell** [comment]
Hello, guys! 

My name is Yoli and I´m the writer. Erik told me about the game, he asked for my support and I help him with the small introducción. If everyone like the story and want that I help you, it´s ok for me. I would like to do the development of the story and I will try to support you with what I may. Well, I am willing to change somethings that you want, to make the story everyone´s taste and look better.

PD: My english is very awful, please keep me a little patience

@KOBUGE-Games/owners

**Comment by mamarilmanson** [comment]
@Etiell ~ nothing much to say but... Hello... :D

**Comment by bojidar-bg** [comment]
@Etiell ~ Well.. like @mamarilmanson I can say nothing else, but welcome :wink:

**Comment by akien-mga** [comment]
The story is pretty nice :-D My only concern is that it focus on spiders, while I thought spiders were just the "first enemy", but we would add many other types of enemies as the game grows. If we go with the spider scenario, we'd need to design different types of spiders, but not too many non-spider enemies – note that it would not be a problem IMO, but we should discuss it ;)

As for @ObaniGemini's suggestion about having a scenario which is a bit more "off-the-wall", basically we have to make a choice now:
- Either we want to go with a serious scenario (the current mockup builds an atmosphere similar to that of [Shadowgrounds](https://en.wikipedia.org/wiki/Shadowgrounds) or Half-Life). With such a scenario, we would go with a somewhat realistic gameplay and level design.
- Or we decide to go with something more light-hearted, not too serious, like in the examples @ObaniGemini gave or e.g. the Commander Keen series (a young boy whose babysitter gets eaten by aliens and who dons his toy equipment to go combat aliens in space, where he meets wise elders and princesses). This would allow to bring more random elements, so any kind of monster or weapon or design layout.

I don't have a preference yet, the two possibilities make for quite different games which could both be pretty cool :) What are your thoughts?

**Comment by bojidar-bg** [comment]
@akien-mga ~ YAY, at last somebody agreed with me that spiders shouldn't be the main focus of the game :smile: 

:+1: for the second idea (light-hearted/not-that-realistic)

**Comment by alketii** [comment]
Spiders were never meant to be the only enemy, if you need more enemies assets, check my exported repo from GC called Violetland.

**Comment by ObaniGemini** [comment]
@bojidar-bg : I always agreed to that idea. I just thought they were kept because the creator needed ennemies and took spiders pictures :|

**Comment by bojidar-bg** [comment]
@ObaniGemini ~ FYI, I was talking about @erik90mx :stuck_out_tongue:

**Comment by erik90mx** [comment]
@KOBUGE-Games/owners

@Etiell gave me this second kind of story to translate and know your opinion:
**STORY 2**
"**_Introduction:**_
The executive was walking in the middle of the city going to his home, insistently looking at his watch, squeezing the handle of his briefcase and avoiding the best he could the amount of people around him. When hitting his shoulder with someone else, he looked up realizing he was in the middle of a meadow. The protagonist was disconcerted by the abrupt change of scenario, in one moment he was in a crowed street, with skyscrapers covering the sky, and now he was alone, with the green grass below his old shoes, with the wind stirring his clothes and the sun blinding him. He turned on his own axis noting that there were three mountains behind the forest that he was able to spot and a path of dirt road beside him.

He pulled his cell phone from the pants pocket, but when opens it he realized that it doesn’t have signal. Without knowing exactly what to do, he began to walk through the path that heading to the forest, saving the hope to found someone that will be able to help him. The sunset caught him while he was reaching the forest entry. There, above a felled tree, he found a gun with a note. The clean paper with scrawls written said: “Use me”. The protagonist takes the gun like it was a wild animal that might bite, realizing with surprise that it was real.    He left it in the same place, along with the note and he look to the forest, without knowing if he should enter or not.

After thinking about it few seconds, he decided to venture into the thick forest. The night fell when he 
had just taken a few steps, the sounds disappears, the air breezes were inexistent and then he stopped; realizing he was surrounded by several pairs or red eyes. Frightened, slowly he walked backward, without daring to turn away from whatever was watching him from the trees. A deafening squawk made that the protagonist dropped his briefcase, to cover his ears, turn around and begin to run to the forest entry, where the gun was. The protagonist lunged at the felled tree, picked up the gun and pointed it toward the creatures that were behind him.

**Here the game starts**:exclamation:

**_General ideas:**_
- “Someone” will guide the protagonist, with notes, through the game. 
- Following the instructions of that “person or thing” doesn't mean that will be always the better choice, the easy one or the right one.
- In some cases it will seems like it is making fun about the protagonist, by leaving him to his own dead or saving him from an imminent danger.
- there will be multiple endings?

**_Extra:**_
If you want, you can use the part in where the protagonist is rounded by giant spiders and with boxes (the repository image), and put something like this for that scene:
For X reason the protagonist arrives to a fenced place, but the door is without a lock and besides it the bomb gun, a lantern and a note that says: “Come to play with us”.
Example of how I imagine the creatures that hunt the protagonist:
http://css.pc.guias-trucos-juegos.com/wp-content/gallery/imagenes-drakensang-2-pc/gargolas.jpg 

**_Story 2 Character Profile:**_
- **Sex:** Men
- **Race/species:** Human
- **Ethnicity:** Asian
- **Age:** 35 years
- **How old they appear:** 1,74 meters y 68 kilograms; Thin; white skin; black, short and straight air; Asian and black eyes. 
- **Occupation:** Lawyer
- **City:** Tokyo
- **A little back story:** He is the only son, his parents are entrepreneurs and all his life try to be the best, no matter how difficult the situation is. Be perfect was the daily normal routine, so after long sleepless nights, achieving his goals and lead a life full of sacrifices, he managed to enter the lawyer’s firm that he wanted. He married at 30 and has 1 year old son. He is a very serious guy, with obtuse mind, patient and astute."

@ObaniGemini  She told me that will send me another story with a more funny proposal too.

**Comment by bojidar-bg** [comment]
@erik90mx ~ This story is also good, but the part with the "teleportation" lacks some basic scientific explanation (what happened? Did he suddenly fell asleep before being transported there? Is it all a dream?)...

**Comment by Etiell** [comment]
First story, or first choice:
1. If you choose the first option then I will change the spiders as the main enemy and will add poisonous animals. 
   1. If you want another plot then you can give me your opinions or ideas to modify it. 

@akien-mga, @ObaniGemini, @alketii.

Second story: 
@bojidar-bg : I imagined three options about why the protagonist teleports
. 
1. Everything is a dream and to awake you need to complete all levels. 
2. The protagonist dies, is in some kind of limbo and to save his soul he has to finish the game. 
3. A demon took the protagonist to that place and is making fun of him.

**Comment by mamarilmanson** [comment]
That no. 3 reminds me of Samurai Jack...

**Comment by ObaniGemini** [comment]
Is that a story or a cinematic ?...

**Comment by erik90mx** [comment]
@ObaniGemini well I was thinking in a mixture of cinematic (still photos with some tween animation to be dynamic) and in game triggers to get the mod of the story in certain areas (like the action areas). The animation system in Godot can be really helpful to get that cinematic animations in the workflow, and we just need to dev the in game story module (maybe the current scene will be paused and text/images over it waiting a Key input to continue)

@Etiell in this moments is editing the third proposal, once she send me that, I will translate it and submit it here, and we will have 3 different proposals to choose one and work it more.

@mamarilmanson made a really great drawing about the first character profile, in this issue:  https://github.com/KOBUGE-Games/ringed/issues/47

**Comment by erik90mx** [comment]
@KOBUGE-Games/owners

@Etiell Gave me the third proposal, here is it:
**STORY 3**
"**_Introduction:**_
There have been strange events on earth: simultaneous mass disappearances and unsolved crimes, but nothing so big to alert the galactic police. The special agent, our protagonist, decided to go to the earth to spend his vacations and, taking advantage to investigate if the things that he heard were true or just rumors to discredit the planet earth. Things were very normal at the beginning; there were crimes of course, but nothing that he doesn't saw before. The strange thing was when he goes through a blind alley and saw how a hamburger, with the size of a dog and sharp teeth, devoured a being from another planet. Things happened so fast, that the agent did not have the chance to do anything before the murderer escape, climbing the walls.

Our protagonist decided not to inform about what happened to his bosses, because he didn't know how to explain it and had no evidence to prove what happened. He returns to the hotel where he was staying and decided begin to investigate the case, however things got complicated. The killer food started to come out from everywhere to attack and devour the helpless citizens of Pelucas city. The special agent takes his weapons and goes out to the streets, willing to help the beings from other planets to survive

**Here the game starts**:exclamation:

**_Explication:**_
After several years in which humans were on the verge of extinction, beings from other planets made a formal appearance on earth, to talk with the leaders and make an offer:  They would clean our planet, would restore our ecosystem and would help the humans; in exchange for permanently live on the planet earth. This was the only permitted offer in how they can “invade” the earth. The universe has very strict laws and if these are not followed to the letter, the galactic policies intervene and nobody likes to go against them.

The restoration was fast, in less than 10 years everything returned to normal and the few humans who still existed; had to adapt to the major changes that were made. The laws needed to be rethinking to benefit both sides and after a while the planet earth eventually became the most famous tourist center between neighbor galaxies. However, some people felt resentful with the beings from other planets, for doing all of these drastic changes in their world, because even the money, racism, social inequality and the big taboos, didn't exist now.

**_General ideas:**_
- Someone is mutating the food to be alive and devour the beings form other planets.
- They create a machine with the purpose that the communications fail and any spaceship can enter or leave the planet earth.
- The terrorist group called The “Taquito”, have *6 subordinates (with food names and dressed like that) that are in strategic points on the earth, to end all the beings from other planets. 
  - _I was talking with Erik and we decided that each of us could do a character that represents the typical food of your country. I don’t know if you like the idea._
- At the beginning the player and the characters in the story think that the main enemy is a being from other planet, but in the end it turns out that the humans were always the true enemies and everything because the hate they have to the “aliens”. They dress in that ridiculous way with the intention that no one discovers their true identity and have no reprisals against humans, in case everything fails.
- If you like the idea, there can be weapons like: bread bombs, forks and knives guns, I don’t know, what do you like to see?

I know that I don't dig enough on some issues, like: how is the world's government now and things like that, the reason is because this story is more about humor, but if you like to hear about that points , or you have doubts please ask me about it.

**_Goals for now:**_
- Kill the mutant food.
- Find weapons and information.
- Fight against each of the 5 subordinates and in the end against the leader. 

**_Story 3 Character Profile:**_
- **Sex:** Male
- **Race/species:** Alien/Dog
- **Ethnicity:** Unknow 
- **Age:** 42 human years
- **How old they appear:** It is an alien race that looks like dogs, but they can speak, walk in two feet and do almost everything that a human can do. His face looks like an adorable puppy, although his true personality is aggressive and feral. He has a scar in the left eye. He is 64 centimeters tall and has a weight according to his body. When is dressing himself normally wear very formal clothes, a weapon in his waist and a pure in his snout.
- **Occupation:** Intergalactic Police
- **A little back story:** He is divorced, has four puppies that see in holydays. Was working for the agency almost 15 years, but because of his horrible character, he is not very popular between his teammates. The agency forced him to take a vacation, because a trouble that he had after knocks out a teammate to mock about his thick and sensual voice.

**_Protagonist References:**_
http://static.globered.com/images/users/126456/2010040701274800000544360000126456.jpg
http://img15.deviantart.net/e4d5/i/2012/207/5/9/peter_puppy_by_edyouassbaka-d58q8d6.png

**Intergalactic Police:**
http://vignette2.wikia.nocookie.net/penguinsofmadagascar/images/d/df/Dancingdogs.png/revision/latest?cb=20120617163009

They don't necessarily need to be dogs, you can use other animal species.

**_Mutant Food:**_
http://www.cinevistablog.com/wp-content/uploads/2013/11/lluvia-de-hamburguesas-2-estreno-3d.jpg
http://i1.wp.com/www.comingsoon.net/nextraimages/cloudywithachancesequelimage3.jpg?resize=514%2C385

I wouldn't like if the mutant food is bigger than a human, but had a median size, like the size of a big dog.

**_Well, these are my three proposals. I like this more, but which of these you like more?**_"

Personally when I heard from Yoli this third proposal, it sounds really good to me. I vote for this one if you ask me :) , it have a really funny game potential.

**Comment by bojidar-bg** [comment]
Ok, @KOBUGE-Games/owners I think we should start a vote... Here is a (minimalistic) form: http://goo.gl/forms/yemkDPbFDU

Note that if you don't want to use google forms (I know some of you that don't ...) you can write a comment to this issue, and ask me to submit it for you :wink: :smiley:

**Comment by erik90mx** [comment]
@bojidar-bg  thanks for the survey! I will comment to Yoli about that.
I voted for 2 and 3
PD: We should share the survey to external people?

**Comment by HarsulinsGhost** [comment]
Hey guys. I just recently got myself a Git Hub account, (or actually my brother made it for me) and I found Ringed, which I think is an appealing open source game idea. I am particularly interested in the story developments thus far and was wondering if I might ask some questions and give my opinions on the story and art elements that I've looked at on this project?

**Comment by Calinou** [comment]
@HarsulinsGhost Sorry, but this project is not actively developed anymore.

**Comment by ObaniGemini** [comment]
Feel free to pursue it by yourself tho

**Comment by HarsulinsGhost** [comment]
Oh wow, really! That's too bad, I thought it might be pretty neat. I don't know that I can do much with it by myself but I might take a hack at it, thanks.  ; )
